src/main/kotlin/com/terraformation/backend/customer/model/ProjectInternalUserModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.terraformation.backend.customer.model
+
+import com.terraformation.backend.db.default_schema.ProjectInternalRole
+import com.terraformation.backend.db.default_schema.UserId
+
+class ProjectInternalUserModel(
+    val userId: UserId,
+    val role: ProjectInternalRole? = null,
+    val roleName: String? = null,
```

Comment:
Maybe add an `init` block to check that exactly one of these is non-null?

Reply:
We have this checked at the DB level as well but I'm always a fan of more being more explicit